Special abilities august 2021 (#25)
GameBrains/er-core: modules/special-rules/pages/supreme-commander.adoc

Where your unit has the {supreme-commander} ability:

* All of your detachments that have their HQ unit within 30 cm of the {supreme-commander} unit gain the {stubborn} ability.

> * If the unit contributes its Assault value to a close combat or contributes its Firepower value to a firefight then you may re-roll a single die when making the combat resolution roll.
> If you re-roll a die then you must abide by the second roll.

Reviewer: I think it'd still make best sense to stick with singular here, because that is the default and the optional alternate method explains itself anyway.
Also, I intend to separate 'close combat' and 'firefight' clearly throughout, and try "Assault resolution" as the more general term. So, I put it that way in this suggestion and I'll tweak others in similar fashion and see how it looks overall.

```suggestion
* If the unit contributes its Assault value to a close combat, or contributes its Firepower value to a firefight, then you may re-roll the die for Assault resolution.
If you re-roll the die then you must abide by the second roll.
```

+
//[TODO: Does "fights in a close combat" include not only base-contact but also 'supporting fire'?]
// IJW editing note: I've changed the text to refer to using the Assault value, based on the original rules's reference to the SC unit itself needing to fight in a CC, not just its bodyguard. I've also tweaked the re-roll text based on the Fate Card Counterattack text, and to allow it to work more easily for players who use highest of 2d6 for combat resolution.

> == Tyranid Dominatrix
> In addition to the above, if the Dominatrix supports a close combat you may re-roll a single die when working out who wins the combat.

Reviewer: @IJW-Wartrader I presume you mean to add content here instead of in the `war-engine-supreme-commander.adoc` file, which we can thus delete. I think that's good.

I'd like to approach it slightly differently it terms of layout, and also: Might we want to make this a thing for all cases where a War Engine has the Supreme Commander ability?

```suggestion
Where your unit has the {supreme-commander} ability and is also a War Engine unit, then in addition to the above:

* If the unit adds 'supporting fire' to a close combat, then you may re-roll a single die when work you roll to resolve the close combat. 

[NOTE:e40k]
.War Engine and {supreme-commander}
====
_Epic 40,000_ only states this additional rule in the case of the Tyranid Dominatrix.
We make it a more general rule -- so it also applies to an Ork Warlord mounted in a Gargant, for example.
====
```

Author: > I'd like to approach it slightly differently it terms of layout, and also: Might we want to make this a thing for all cases where a War Engine has the Supreme Commander ability?

I suggested that on FB a while ago, and it was... _unpopular_...

Reviewer: Ah, okay. In that case, something like this:

```suggestion
[NOTE]
.Tyranid Dominatrix -- additional effect for {supreme-commander}
====
The Tyranid Dominatrix unit is a special case.
Not only does it have the {supreme-commander} ability, but for this unit only the ability has this additional effect:

* If the unit adds 'supporting fire' to a close combat, then you may re-roll a single die when work you roll to resolve the close combat. 
====
```
It makes rather an odd exception of course. Given that it is specific to one unit, it'd probably be best to move this rule to the actual Tyranid Dominatrix datasheet when we get to it.

**Aside**: We may also need to tighten up use of terms like 'supporting fire' in due course, to be 100% clear about when we mean that a rule applies where the unit supports a close combat where its own detachment is engaged (our main use of 'supporting fire'), and when we mean that a rule does or does not apply where the unit 'supports' a friendly detachment in a separate close combat. But I think that we leave this as a discrete task to tackle later.

Author: > It makes rather an odd exception of course. Given that it is specific to one unit, it'd probably be best to move this rule to the actual Tyranid Dominatrix datasheet when we get to it.

I think it still needs to be in the rules as well, I just didn't want to add an extra ability and page for something that only affects one single unit. :-(

> **Aside**: We may also need to tighten up use of terms like 'supporting fire' in due course, to be 100% clear about when we mean that a rule applies where the unit supports a close combat where its own detachment is engaged (our main use of 'supporting fire'), and when we mean that a rule does or does not apply where the unit 'supports' a friendly detachment in a separate close combat. But I think that we leave this as a discrete task to tackle later.

Agreed, but in this case the term we're currently using in the War Engine section is 'support', rather than 'supporting fire'.

Would this work?
```suggestion
[NOTE]
.Tyranid Dominatrix -- additional effect for {supreme-commander}
====
The Tyranid Dominatrix unit is a special case.
Not only does it have the {supreme-commander} ability, but for this unit only the ability has this additional effect:

* If the unit supports another detachment that is in close combat, then you may re-roll a single die when work you roll to resolve the close combat. 
====
```

Reviewer: I'd misapprehended a point I won't bore you with, but I can see where I'd gone wrong now. Yes, given the actual situation and given our current use of terms, I agree that we should not use 'supporting fire' in this case.

I've though of some further tweaks though -- in part to clarify some more, and in part to mirror the latest language in the earlier lines.

```suggestion
[NOTE]
.Tyranid Dominatrix -- additional effect for {supreme-commander}
====
The Tyranid Dominatrix unit is a special case.
Not only is it a War Engine that has the {supreme-commander} ability, but for this unit only the ability has this additional effect:

* If the Dominatrix supports another detachment that is in close combat, then you may re-roll the die for Assault Resolution in that close combat.
If you re-roll the die then you must abide by the second roll.
====
```

Author: Looks good to me!

Reviewer: Cool. I'll commit this, and make time soon to build and make final checks with a view to approve/merge/publish. :)

* If the unit contributes its Assault value to a close combat, or contributes its Firepower value to a firefight, then you may re-roll the die for Assault resolution.
If you re-roll the die then you must abide by the second roll.
+
//[TODO: Does "fights in a close combat" include not only base-contact but also 'supporting fire'?]
// IJW editing note: I've changed the text to refer to using the Assault value, based on the original rules's reference to the SC unit itself needing to fight in a CC, not just its bodyguard. I've also tweaked the re-roll text based on the Fate Card Counterattack text, and to allow it to work more easily for players who use highest of 2d6 for combat resolution.

[NOTE]
.Tyranid Dominatrix -- additional effect for {supreme-commander}
====
The Tyranid Dominatrix unit is a special case.
Not only is it a War Engine that has the {supreme-commander} ability, but for this unit only the ability has this additional effect:

* If the Dominatrix supports another detachment that is in close combat, then you may re-roll the die for Assault Resolution in that close combat.
If you re-roll the die then you must abide by the second roll.
====

> 

Reviewer: ```suggestion
```
